packages/fast-tooling-react/package.json

```diff
@@ -66,6 +66,7 @@
     "@babel/preset-env": "^7.1.0",
     "@babel/preset-react": "^7.0.0",
     "@microsoft/fast-tslint-rules": "^3.0.4",
+    "@types/fast-json-stable-stringify": "^2.0.0",
```

Comment:
I think we can remove this, we will want our package to break if `ajv` removes this as a dependency, but it will pass if we leave this here.

Reply:
fast-json-stable-stringify dependency doesn't come with the typescript types so this one is just to enable safe typing while using this dependency. It is in the dev dependency list so it won't affect production bundle. I didn't include fast-json-stable-stringify itself and code will break if ajv removes it.